app/src/main/java/net/skyscanner/backpack/demo/meta/ViewStory.kt

```diff
@@ -0,0 +1,39 @@
+package net.skyscanner.backpack.demo.meta
+
+import android.content.res.Configuration
+import android.graphics.Color
+import androidx.annotation.LayoutRes
+import androidx.compose.ui.tooling.preview.Preview
+import javax.annotation.concurrent.Immutable
+
+@Immutable
+@Suppress("Detekt.PreviewNaming")
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.BINARY)
+
+@Preview(name = "Default",
```

Comment:
We already using multiple preview annotations. It's just these annotations themselves are duplicated for View and Compose.

We can create something like DmPreview, RtlPrevew, etc, but I'd rather not create new entities – these annotations can be used for any other composable. A solution would be to extract to to a different module and make internal, but it's a bit overkill

Reply:
yeah can we create a shared definition for all three and then reuse it across view + compose. this is also something that we may want to have in backpack, rather than just here, so it can be utilised in the app as well.

also can we use the canvas colours instead of just black/white?